src/components/AnonymousChat/child/ChatItemMyTextV2.tsx

```diff
@@ -132,7 +142,8 @@ const ChatItemMyTextV2 = ({
             previewBackgroundColor="transparent"
             style={{flex: 1}}
             actions={contextMenuActions}
-            onPress={(e) => onContextMenuPressed(e, data, chatType)}>
+            onPress={(e) => onContextMenuPressed(e, data, chatType)}
+            disabled={attachments.length > 1}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It's good to see that you've added a condition to disable the context menu when there are more than one attachments. This will prevent users from performing actions on multiple attachments at once, which could lead to unexpected behavior.



<!-- This is an auto-generated comment by OSS CodeRabbit -->